packages/server/src/sdk/app/rows/search.ts

```diff
@@ -122,9 +122,8 @@ export async function fetchRaw(tableId: string): Promise<Row[]> {
 }
 
 export async function fetchView(
-  tableId: string,
   viewName: string,
   params: ViewParams
 ): Promise<Row[]> {
-  return pickApi(tableId).fetchView(viewName, params)
+  return internal.fetchView(viewName, params)
```

Comment:
Should this always go to the internal API?

Reply:
I just renamed it.
It is for the legacy views. We don't have external legacy views. It was working fine as the current code checks if the provided table id is external to pick the external sdk. Otherwise, it uses the internal. Because the table id was undefined on this controller, it did always fall-back to internal.